lib/src/main/java/io/ably/lib/http/Http.java

```diff
@@ -325,7 +327,17 @@ public void finalize() {
 	 * @throws AblyException
 	 */
 	<T> T ablyHttpExecute(String path, String method, Param[] headers, Param[] params, RequestBody requestBody, ResponseHandler<T> responseHandler) throws AblyException {
-		int retryCountRemaining = Hosts.isRestFallbackSupported(this.host) ? options.httpMaxRetryCount : 0;
+		List<String> customFallbackHosts = null;
+		int retryCountRemaining = 0;
+		if (options.fallbackHosts != null) {
+			if (options.fallbackHosts.length > 0) {
```

Comment:
Unless I am reading this incorrectly, having `options.fallbackHosts` with an empty array is now going to use the defaults instead of disabling fallback host functionality?  Have I misread it?